app/src/main/res/values-af-rZA/strings.xml

```diff
@@ -179,6 +179,8 @@
     <item>QIF</item>
     <item>OFX</item>
     <item>XML</item>
+    <item>CSV-accounts</item>
```

Comment:
The translation is handled by the CrowdIn platform. No need to do it in the translation files. 
I should add this to the development guidelines.

Reply:
I'll try to fix it in 5-10 days